spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -71,6 +71,14 @@
             <optional>true</optional>
         </dependency>
 
+        <!-- Pub/Sub Lite -->
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>spring-cloud-gcp-pubsublite</artifactId>
+            <version>2.0.3-SNAPSHOT</version>
```

Comment:
The version should be pulled in automatically, so you shouldn't need to specify here.

Reply:
It causes build errors if I don't, I'm unsure why.

Reply:
You'll have to add pubsublite as an entry to [spring-cloud-gcp-dependencies](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/spring-cloud-gcp-dependencies/pom.xml#L89) so that the version gets registered. It should allow you to remove hard-coding the version here after that.